### ISSUE TITLE
Neurodamus incompatible with pyneuron

### ISSUE THREAD

**Issue by wvangeit** [BlueBrain/spack#375]
While discussing with @pramodk how https://github.com/BlueBrain/spack/issues/366 can be solved we identified that the neurodamus spack packages don't seem to be compatible with pyneuron.

E.g. the following trivial script fails:
```
module load neurodamus-hippocampus/0.2                                                                                                                                                                                                                                                                                                                                                                     
python -c 'import neuron'      
```
with
```
MPT ERROR: mpiexec_mpt must be used to launch all MPI applications
```

**Comment by pramodk** [comment]
Being addressed by #371 : 

```
kumbhar@bbpv2:~$ unset MODULEPATH
kumbhar@bbpv2:~$ source  /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/371/config/modules.sh
kumbhar@bbpv2:~$ module load neurodamus-hippocampus/0.2 py-bglibpy/4.0.17/python3
Autoloading reportinglib/2.5.1
Autoloading hpe-mpi/2.16
Autoloading python/3.6.5
Autoloading neuron/7.6.6/python3/parallel

kumbhar@bbpv2:~$ python -c 'import neuron'
kumbhar@bbpv2:~$
```

Will merge that once we fix another issue we saw : `undefined symbol: __libm_exp_table_128`

**Comment by ferdonline** [comment]
@wvangeit closing this issue for now. If you find any issue please reopen
